src/devices/src/virtio/rng/device.rs

```diff
@@ -237,7 +292,10 @@ impl VirtioDevice for Entropy {
 
 #[cfg(test)]
 mod tests {
+    use std::time::Duration;
+
     use super::*;
+    use crate::check_metric_after_block;
```

Comment:
everytime I see this function I have to do a double take because I think its related to the block device 😂